clients/repo_client.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clients
+
+type RepoClient interface {
+	InitRepo(owner, repo string) error
+	GetTarball() []byte
```

Comment:
nit: since we are creating interface, might be nice to use generic name like GetRepoArchive. when you expand to gitlab or others, archive can be tar, zip or whatever. same for client.targz

Reply:
Done.